messaging/models.py

```diff
@@ -42,7 +35,15 @@ class Message(TimestampedModel, TranslatableModel):
             _("Subscribed to free spot notification"),
         ),
     )
-
+    EMAIL = "email"
+    SMS = "sms"
+    PROTOCOL_CHOICES = ((EMAIL, _("Email")), (SMS, _("SMS")))
+    protocol = models.CharField(
+        max_length=64,
```

Comment:
Just on observation that 64 seems like quite much when the current maximum length will be 5. Maybe 8 or 16? :sweat_smile: